src/main/java/com/blamejared/crafttweaker/impl/recipes/replacement/ReplacerAction.java

```diff
@@ -62,9 +68,7 @@ public String describe() {
                         .map(ResourceLocation::toString)
                         .collect(Collectors.joining(", ", "recipes {", "}")),
                 this.rules.stream().map(IReplacementRule::describe).collect(Collectors.joining(", ", "{", "}")),
-                this.exclusions.isEmpty()? "" : this.exclusions.stream()
-                        .map(ResourceLocation::toString)
-                        .collect(Collectors.joining("\", \"", " excluding {\"", "\"}"))
+                this.joinBothLists(this.userExclusions, this.defaultExclusions)
```

Comment:
Adding this makes the line become even longer than it already is. Would it be better to split it into multiple lines? If so, which format would be better?